apps/03TicTacToe.js

```diff
@@ -22,24 +22,74 @@ function printBoard() {
 }
 
 function horizontalWin() {
-    // Your code here
-}
+  if (board[0][0] === playerTurn && board[0][1] === playerTurn && board[0][2] === playerTurn) {
+    return true;
+  }
+  if (board[1][0] === playerTurn && board[1][1] === playerTurn && board[1][2] === playerTurn) {
+    return true;
+  }
+  if (board[2][0] === playerTurn && board[2][1] === playerTurn && board[2][2] === playerTurn) {
+  return true;
+  }
+  else {
+      return false;
```

Comment:
This is legit, but it looks like you were thinking of an if/else if/else structure. 

Tying the else to just the last `if` works in this case, but it isn't perfect logic (always striving for perfection :D)